### DISCUSSION TITLE
Problem with Playergroup

### DISCUSSION THREAD

**Discussion by philadou** [music-assistant/support#3168] [Q&A] [answered]
Hello everyone. Somehow I’m making a mistake in my thinking and I can’t get any further…

I’m using MA in the current version. 
I’m using 4 HomePods via AirPlay as playback devices. 
I’ve also created an “Everywhere” AirPlay player group that includes all my 4 HomePods.

<img width="365" alt="img1" src="https://github.com/user-attachments/assets/e4e4a9b4-105b-45a0-9655-8efba1b7b788">

Now, I play a playlist on my “Everywhere” player group. Up to this point, everything is fine.

But when I turn off any one of the HomePods, suddenly all the other HomePods stop playing. The others should still continue playing music, shouldn’t they? Or am I thinking wrong??

I would appreciate any help.
Cheers,
Phil

**Comment by marcelveldt** [comment, marked as answer]
If you are playing to a group, you are playing to a group. Turning off a member would then "disrupt" that group you defined.
If you want to stop playing on one of the devices of a group you have several options:

1) transfer playback to a smaller group
2) mute playback on that player you want to not play audio
3) Enable dynamic members feature in the player group to allow you to dynamically update the member list.
4) Don't use a group but manually sync players to one a player (and use join/unjoin)

**Reply by philadou** [reply]
Thank you very much for the great and helpful answer 👍🏻

Option 2:
That's exactly how I have it set up at the moment. I set the volume to 0 on the HomePods via automation. However, strangely, after an irregular amount of time, I suddenly hear music very faintly coming from the "off" HomePods. It doesn’t seem to work reliably for me.

Option 3:
That sounds great. When I go to configure my player group, I can’t find any setting called "dynamic members." Where can I activate this feature?

**Reply by OzGav** [reply]
Option 3 is only available to permanent sync groups. https://music-assistant.io/faq/groups/#sync-groups
